docs/DeveloperGuide.md

```diff
@@ -1072,11 +1100,64 @@ _{More to be added}_
   - 1a1. Return an error indicating command not recognised and provides the correct command format.
   - Use case ends.
 - 1b. Additional arguments are specified after the command.
-  - 1b1. System will ignore those arguments and execute the comamnd as usual.
+  - 1b1. System will ignore those arguments and execute the command as usual.
 - 2a. There are no existing classes.
   - 2a1. System will return a message indicating that there are no existing classes in the list.
     <br>
 
+#### Use case 8: Allocate Student to tutorial team
+
+**Actor**: User
+
+**System**: TAHelper
+
+**MSS**
+1. User specifies the student id of the student, the module code, tutorial class, and the tutorial team name
+in the tutorial class to allocate the student into.
+2. User enters the command and along with the details.
+3. System allocates student into tutorial team and displays a message. 
+   Use case ends.
+
+**Extensions**
+
```

Comment:
Do we need to put invalid input format(s) in the extensions (as part of step 1)?

Reply:
Yes invalid input handling should be placed in the extension instead of MSS